src/javascript/app/pages/user/set_currency.js

```diff
@@ -109,11 +109,15 @@ const SetCurrency = (() => {
                     text: Currency.getCurrencyName(c) || c,
                     ...(/^UST$/.test(c) && {
                         'data-balloon'       : localize('Tether Omni (USDT) is a version of Tether that\'s pegged to USD and is built on the Bitcoin blockchain.'),
-                        'data-balloon-length': 'large',
+                        'data-balloon-length': 'medium',
+                        'data-balloon-pos'   : 'top',
+                        'class'              : 'show_mobile',
```

Comment:
I think we use `-` as a separator for multi-word class names in binary-static.

Reply:
I utilized the existing class created 3 years ago actually, do I change it all over the code? It has 26 occurrences  across 13 files.

Reply:
Ah no need to do so.